### ISSUE TITLE
Cannot add a file larger than 2 GB

### ISSUE THREAD

**Issue by igoticecream** [TeamShinkansen/Hakchi2-CE#17]
Im trying to add FFVIII (2.2GB .PBP) but hakchi2 cannot process it

```
The file is too long. This operation is currently limited to supporting files less than 2 gigabytes in size.   at System.IO.File.ReadAllBytes(String path)
   at com.clusterrr.hakchi_gui.NesMiniApplication.Import(String inputFileName, String originalFileName, Byte[] rawRomData) in C:\hakchi2\Apps\NesMiniApplication.cs:line 416
   at com.clusterrr.hakchi_gui.WorkerForm.AddGames(IEnumerable`1 files, Form parentForm) in C:\hakchi2\WorkerForm.cs:line 1949
The file is too long. This operation is currently limited to supporting files less than 2 gigabytes in size.   at System.IO.File.ReadAllBytes(String path)
   at com.clusterrr.hakchi_gui.NesMiniApplication.Import(String inputFileName, String originalFileName, Byte[] rawRomData) in C:\hakchi2\Apps\NesMiniApplication.cs:line 416
   at com.clusterrr.hakchi_gui.WorkerForm.AddGames(IEnumerable`1 files, Form parentForm) in C:\hakchi2\WorkerForm.cs:line 1949
```

Is there a workaround in the meanwhile?

**Comment by daPhie79** [comment]
we have to find a workflow that prevents huge files from being read.

**Comment by daPhie79** [comment]
i fixed this issue, will be released in 1.2!
